src/Lib/Mailer/Headers.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Lib\Mailer;
+
+class Headers
+{
+    /** @var array<string|array> */
+    private $headers;
+
+
+    public function __construct()
+    {
+        $this->setHeader('Bcc', [])
+             ->setHeader('Content-Type', 'text/html; charset=utf-8')
+             ->setHeader('X-Mailer', 'PHP/' . phpversion());
+    }
+
+
+    /**
+     * Set single header
+     *
+     * @param string $key
+     * @param mixed $content
+     * @return self
+     */
+    private function setHeader(string $key, $content): self
+    {
+        $this->headers[$key] = $content;
+        return $this;
+    }
+
+    
+    /**
+     * Get single header
+     *
+     * @param string $key
+     * @return mixed
+     */
+    public function getHeader(string $key)
+    {
+        return $this->headers[$key];
+    }
+
+
+    /**
+     * Return string header for email
+     *
+     * @return array<string|array>
```

Comment:
Should return `array<string>`.

Reply:
![image](https://user-images.githubusercontent.com/47446443/105736394-06fbc200-5f35-11eb-97be-f10a6cd2dfc8.png)
php satan has problems with that

Reply:
Right, so keep it as it is.